src/OperatingSystemApp/oscursormodel.js

```diff
@@ -110,8 +74,8 @@ var OSCursorModel = GObject.registerClass({
          * up to 256px
          */
         this.theme = 'cursor-default';
-
-        this._touchpadSettings.reset('speed');
+        this.size = 24;
+        this.speed = 0;
```

Comment:
It would be nice if this was using a constant defined somewhere, but we have a big list of magic numbers with all the `GObject.ParamSpec` calls here as well so :man_shrugging: